clang/test/Sema/warn-extra-cast-function-type-strict.c

```diff
@@ -0,0 +1,42 @@
+// RUN: %clang_cc1 %s -fsyntax-only -Wextra -verify
+
+
+int t(int array[static 12]);
+int u(int i);
+const int v(int i); /* expected-warning {{'const' type qualifier on return type has no effec}} */
```

Comment:
```suggestion
const int v(int i); /* expected-warning {{'const' type qualifier on return type has no effect}} */
```

Reply:
Didn't have to do it as the file is now not added.